run

```diff
@@ -177,8 +204,8 @@ case $run_command in
 
     # Run watch command in the background
     if [[ ${1:-} == "--watch" ]] || [[ ${1:-} == "-w" ]]; then
-      container_name="yarn-$(date +'%s')"
-      trap 'docker kill ${container_name}' EXIT
```

Comment:
Oh I can see how it could have been erroring. Because `${container_name}` gets parsed at the point of `ctrl+c` rather than on instantiation, if its value has changed then it will kill the wrong thing. My testing probably never hit that particular state. Good catch.